bento/src/main/java/com/yelp/android/bento/core/Component.java

```diff
@@ -184,17 +188,6 @@ public int getNumberLanes() {
         return 1;
     }
 
-    /**
-     * Returns the number of lanes at a given position. This will usually return the same as
-     * {@link Component#getNumberLanes()}, but {@link ComponentGroup}s will want to return the
-     * number of lanes at a given index. For example, if you have a {@link ComponentGroup} with
-     * two {@link Component}s with a different number of lanes, this should return the number of
-     * lanes in the component that owns the position.
-     */
-    public int getNumberLanesAtPosition(int position) {
```

Comment:
Eh, a little unrelated, but still related. I find that both grids and reordering require me to be able to find a component at a given absolute index. I just created a getLowestComponentAtIndex, then call whatever method from there. Calling `getLowestComponentAtIndex(n).getNumberLanes()` seems cleaner than having a bunch of redundant methods that recursively find the lowest component, then return some field from it.

Reply:
Gotcha. I don't like the term "lowest" to refer to the leaf node however. Also, since this is not just returning a field, I think we should change to "find" instead of "get" so it doesn't give the impression that this is a costless operation. What do you think about renaming it to something like `findIndexOwner` or `findComponentWithIndex`?